examples/uidl-samples/cms-project.json

```diff
@@ -0,0 +1,3684 @@
+                    }
+                  },
+                  {
+                    "type": "element",
+                    "content": {
+                      "elementType": "contentful-richtext-node",
+                      "referencedStyles": {},
+                      "attrs": {
+                        "richText": {
+                          "type": "dynamic",
+                          "content": {
+                            "id": "536f3717-684e-4d58-816f-29975cc9bc93",
+                            "referenceType": "ctx",
+                            "path": [
+                              "content"
+                            ]
+                          }
+                        }
+                      },
+                      "style": {
+                        "width": {
+                          "type": "static",
+                          "content": "100%"
+                        },
+                        "alignSelf": {
+                          "type": "static",
+                          "content": "stretch"
+                        }
+                      },
+                      "children": []
+                    }
+                  }
+                ]
+              }
+            }
+          ]
+        }
+      },
+      "name": "BlogPostDetailsContent",
+      "styleSetDefinitions": {}
+    }
+  },
+  "resources": {
+    "rootFolder": "pages/api",
+    "items": {
+      "7816efdd-2f6b-46f5-b5c6-a4336510c92f": {
+        "id": "7816efdd-2f6b-46f5-b5c6-a4336510c92f",
+        "name": "Home",
+        "baseUrl": {
+          "type": "env",
+          "value": "CMS_URL"
+        },
+        "authToken": {
+          "type": "env",
+          "value": "CMS_ACCESS_TOKEN"
+        },
+        "urlParams": {
+          "content_type": {
+            "type": "static",
+            "content": "book"
+          },
+          "sys.id": {
+            "type": "static",
+            "content": "1nuIm7L8jAzM1wWoCuFS6y"
+          }
+        },
+        "route": {
+          "type": "static",
+          "value": "entries"
+        }
+      }
+    }
+  },
+  "contexts": {
+    "rootFolder": "contexts",
+    "items": {
+      "cb964157-cea2-40bb-b35a-9ec4db6873ba": {
+        "name": "Home",
+        "fileName": "Home"
+      },
+      "d8960784-d161-466a-a2d2-ec08f6243a09": {
+        "name": "Book",
+        "fileName": "Book"
+      },
+      "7a35cbce-5704-4657-b36f-725f79dcba5d": {
+        "name": "BlogPost",
+        "fileName": "BlogPost"
+      },
+      "59798d55-52f0-4e97-a7bb-38fe956252d1": {
+        "name": "Author",
+        "fileName": "Author"
+      },
+      "d2507429-cf7e-4151-b6ff-c22d3e90dedf": {
+        "name": "Author",
+        "fileName": "Author"
+      },
+      "bea417dd-a288-41cc-9c75-df5fe292ba2a": {
+        "name": "Book",
+        "fileName": "Book"
+      },
+      "536f3717-684e-4d58-816f-29975cc9bc93": {
+        "name": "BlogPost",
+        "fileName": "BlogPost"
+      }
+    }
+  }
```

Comment:
This is an option as well, we only had an internal discussion about it. We could do it, but it would mean that the implementation is hidden from the code. We need to decide if this is what we want to do or not. As a developer, I would like to see what and how the fetch request is done, not to be hidden from me; this is a subjective opinion